src/Symfony/Component/Console/Completion/CompletionInput.php

```diff
@@ -53,7 +53,7 @@ public static function fromString(string $inputStr, int $currentIndex): self
      * Create an input based on an COMP_WORDS token list.
      *
      * @param string[] $tokens       the set of split tokens (e.g. COMP_WORDS or argv)
-     * @param          $currentIndex the index of the cursor (e.g. COMP_CWORD)
+     * @param int      $currentIndex the index of the cursor (e.g. COMP_CWORD)
```

Comment:
this is by design. The type is optional and already given in de PHP code. We've done this in multiple locations (and I would even be in favor of removing all duplicate types from the PHPdoc)

Reply:
PHPStorm was complaining about it:
![CleanShot 2024-01-03 at 14 43 40@2x](https://github.com/symfony/symfony/assets/995707/8e1a43ba-343a-4810-ac6a-357f9c932878)

Reply:
I think we should be content, in 5.4 only two more needs to be fixed, which I've done in this PR.
I also created a PR for 6.3
* #53391 

If both are accepted and (up)merged, I will check 6.4, 7.0 and 7.1

Reply:
To me, that's a bug in PHPstorm.

Reply:
@wouterj Do you have a link to a discussion about this? 
I consider such doc param to be incomplete and less readable: when reading a param's description, not having the type right before it requires to find and look at the corresponding parameter inside the signature to get the full picture

Reply:
Sure, it has always been part of our coding standard:

> - Add PHPDoc blocks for classes, methods, and functions only when they add relevant information that does not duplicate the name, native type declaration or context (e.g. instanceof checks);

Discussed in the PR making the change: https://github.com/symfony/symfony/pull/41912

See also https://github.com/symfony/symfony/issues/46140 https://github.com/symfony/symfony/pull/46218 https://github.com/symfony/symfony/pull/46217 https://github.com/symfony/symfony/pull/44935 https://github.com/symfony/symfony/pull/45324

Reply:
Fair enough :)